rpc/wrtc_signaling_answerer.go

```diff
@@ -331,12 +331,12 @@ func (ans *webrtcSignalingAnswerer) answer(client webrtcpb.SignalingService_Answ
 		exchangeCtx, exchangeCancel = context.WithTimeout(ans.closeCtx, getDefaultOfferDeadline())
 	}
 
-	errCh := make(chan interface{})
+	errCh := make(chan error)
 	defer exchangeCancel()
-	sendErr := func(err interface{}) {
+	sendErr := func(err error) {
 		select {
 		case <-exchangeCtx.Done():
-		case errCh <- err:
+		case errCh <- filterEOF(err, ans.logger):
```

Comment:
Won't this send `nil` to `errCh`? Do we want that?

Reply:
yeah probably not, I think this would close the server / client channel, channels are still a bit foreign to me